api/src/main/java/com/lightbend/lagom/javadsl/api/ConfigurationServiceLocator.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.api;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigException;
+import play.Configuration;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+
+/**
+ * A service locator that uses static configuration.
+ */
+@Singleton
+public class ConfigurationServiceLocator implements ServiceLocator {
+
+    private final Map<String, URI> services;
+
+    @Inject
+    public ConfigurationServiceLocator(Configuration configuration) {
+        services = new HashMap<>();
+        if (configuration.underlying().hasPath("lagom.services")) {
```

Comment:
Small point, but since the key `lagom.services` is used three times in this impl, it may make sense to extract a variable.